packages/node_modules/samples/browser-meetings/app.js

```diff
@@ -83,3 +83,117 @@ function unregister() {
         'Not Registered';
     });
 }
+
+// Meetings Management Section --------------------------------------------------
+
+const createMeetingDestinationElm = document.querySelector('#create-meeting-destination');
+const createMeetingActionElm = document.querySelector('#create-meeting-action');
+const meetingsJoinDeviceElm = document.querySelector('#meetings-join-device');
+const meetingsJoinPinElm = document.querySelector('#meetings-join-pin');
+const meetingsJoinModeratorElm = document.querySelector('#meetings-join-moderator');
+const meetingsListCollectElm = document.querySelector('#meetings-list-collect');
+const meetingsListElm = document.querySelector('#meetings-list');
+const meetingsAddMediaElm = document.querySelector('#meetings-add-media');
+const meetingsLeaveElm = document.querySelector('#meetings-leave');
+const meetingsCurrentDetailsElm = document.querySelector('#meetings-current-details');
+
+function generateMeetingsListItem(meeting) {
+  const itemElm = document.createElement('div');
+  const joinElm = document.createElement('button');
+  const detailsElm = document.createElement('label');
+
+  itemElm.id = `meeting-list-item-${meeting.id}`;
+  itemElm.key = meeting.id;
+
+  joinElm.onclick = () => joinMeeting(meeting.id);
+  joinElm.type = 'button';
+  joinElm.value = meeting.id;
+  joinElm.innerHTML = 'meeting.join()';
+
+  detailsElm.innerHTML = meeting.destination ||
+    meeting.sipUri ||
+    meeting.id;
+
+  itemElm.appendChild(joinElm);
+  itemElm.appendChild(detailsElm);
+
+  return itemElm;
+}
+
+function collectMeetings() {
+  console.log('MeetingsManagement#collectMeetings()');
+
+  webex.meetings.syncMeetings()
+    .then(() => new Promise((resolve) => {
+      setTimeout(() => resolve(), 200);
+    }))
+    .then(() => {
+      console.log('MeetingsManagement#collectMeetings() :: successfully collected meetings');
+      const meetings = webex.meetings.getAllMeetings();
+
+      if (Object.keys(meetings).length === 0) {
+        meetingsListElm.innerHTML = 'There are currently no meetings to display';
+
+        return;
+      }
+
+      meetingsListElm.innerHTML = '';
+
+      Object.keys(meetings).forEach(
+        (key) => {
+          meetingsListElm.appendChild(
+            generateMeetingsListItem(meetings[key])
+          );
+        }
+      );
+    });
+}
+
+function createMeeting() {
+  webex.meetings.create(createMeetingDestinationElm.value)
+    .then((meeting) => {
+      if (meetingsListElm.childElementCount === 0) {
+        meetingsListElm.innerHTML = '';
+      }
+
+      meetingsListElm.appendChild(
+        generateMeetingsListItem(meeting)
+      );
+    });
+}
+
+function joinMeeting(meetingId) {
+  const meeting = webex.meetings.getAllMeetings()[meetingId];
```

Comment:
If this fails, the code won't be able to access the index of the array [meetindId]. Can we add some conditionals of try/catch to avoid this?

Reply:
`getAllMeetings()` always returns an object. I added error handling in cases that the meetingId is invalid or no longer exists.